detector/tracker/DriftChamber_o1_v02.cpp

```diff
@@ -75,6 +77,7 @@ static dd4hep::Ref_t create_DCH_o1_v02(dd4hep::Detector &desc, dd4hep::xml::Hand
     }
 
     bool debugGeometry = detElem.hasChild(_Unicode(debugGeometry));
+    bool useG4TT = detElem.hasChild(_Unicode(useG4TT));
```

Comment:
Hi, do you think it is worth adding a comment in the geometry section explaining what this does and how to set it? or better is users do not try it out?

Reply:
it is better users do not try